test.sh

```diff
@@ -30,109 +30,119 @@ function setup_osbs() {
   OSBS_CLIENT_REPO=${OSBS_CLIENT_REPO:-https://github.com/containerbuildsystem/osbs-client}
   OSBS_CLIENT_BRANCH=${OSBS_CLIENT_BRANCH:-master}
 
-  # Pull fedora images from registry.fedoraproject.org
-  if [[ $OS == "fedora" ]]; then
-    IMAGE="registry.fedoraproject.org/$IMAGE"
-  fi
-
   # PIP_PREFIX: osbs-client provides input templates that must be copied into /usr/share/...
   ENVS='-e PIP_PREFIX=/usr'
   RUN="$ENGINE exec -i ${ENVS} $CONTAINER_NAME"
-  PYTHON="python$PYTHON_VERSION"
-  if [[ $OS == "fedora" ]]; then
-    PIP_PKG="python$PYTHON_VERSION-pip"
-    PIP="pip$PYTHON_VERSION"
-    PKG="dnf"
-    ENABLE_REPO="--enablerepo=updates-testing"
-    PKG_EXTRA="dnf-plugins-core desktop-file-utils flatpak ostree skopeo python$PYTHON_VERSION-libmodulemd glibc-langpack-en"
-    BUILDDEP="dnf builddep"
-    PYTHON="python$PYTHON_VERSION"
-  else
-    PIP_PKG="python-pip"
+
+  if [[ $OS == "centos" ]]; then
+    PYTHON="python"
+    PIP_PKG="$PYTHON-pip"
     PIP="pip"
     PKG="yum"
     ENABLE_REPO=
-    PKG_EXTRA="yum-utils epel-release git-core desktop-file-utils flatpak ostree skopeo python2-libmodulemd2"
+    PKG_EXTRA="yum-utils git-core desktop-file-utils flatpak ostree skopeo ${PYTHON}2-libmodulemd2"
     BUILDDEP="yum-builddep"
-    PYTHON="python"
+  else
+    PYTHON="python$PYTHON_VERSION"
+    PIP_PKG="$PYTHON-pip"
+    PIP="pip$PYTHON_VERSION"
+    PKG="dnf"
+    ENABLE_REPO="--enablerepo=updates-testing"
+    PKG_EXTRA="dnf-plugins-core desktop-file-utils flatpak ostree skopeo $PYTHON-libmodulemd glibc-langpack-en $PYTHON-pylint"
+    BUILDDEP="dnf builddep"
   fi
 
-  PIP_INST=("$PIP" install --index-url "${PYPI_INDEX:-https://pypi.org/simple}")
+  # Common install dependencies
+  PKG_COMMON_EXTRA="git gcc krb5-devel python-devel popt-devel"
+  PKG_EXTRA="$PKG_EXTRA $PKG_COMMON_EXTRA"
+
+  PIP_INST=("$PIP" install --index-url https://pypi.org/simple)
```

Comment:
I think we want to keep `"${PYPI_INDEX:-https://pypi.org/simple}"`, @tkdchen added this with the reasoning that specifying a PyPI mirror might make local tests faster for some people

Reply:
ok